src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
```

Comment:
Your calendar must be centered on the screen. Add styles to parent element to get this.